Deployment script error handling docs (#1713)
foundry-rs/starknet-foundry: CHANGELOG.md

#### Changed

- fixed a bug where a profile was passed to scarb even when it did not exist

> - added error handling in deployment scripts (`declare`, `deploy`, `call`, `invoke` now return `Result<T, ScriptCommandError>`)

Reviewer: ```suggestion
- error handling from inside deployment scripts is now possible (`declare`, `deploy`, `call`, `invoke` now return `Result<T, ScriptCommandError>`)
```
